python-backend/app/api/mines/mine/models/mine_commodity_code.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime
+
+from ....utils.models_mixins import AuditMixin, Base
+from app.api.constants import COMMODITY_CODES_CONFIG, METALS_AND_MINERALS
+from app.extensions import db
+
+
+class MineCommodityCode(AuditMixin, Base):
+    __tablename__ = 'mine_commodity_code'
+    mine_commodity_code = db.Column(db.String, nullable=False, primary_key=True)
+    description = db.Column(db.String, nullable=False)
+    active_ind = db.Column(db.Boolean, nullable=False, default=True)
```

Comment:
default values should be set by the db. you can do that by adding this import: 
from sqlalchemy.schema import FetchedValue

then going. 
active_ind = db.Column(db.Boolean, server_default=FetchedValue())

Reply:
Sure. The only defaults I've come across in the codebase so far are timestamps, which have not been using fetchedValue. I can switch this default to use that

Reply:
We haven't done a full refactor of the default values, but I have added as a backlog item and think we should avoid defining defaults in two places. active_ind is a safe one to always be 'true' but default date and guid policies might change in the future so using FetchedValue makes that much easier. Sounds good.